### PR TITLE
fix: delete and update

### PR DESCRIPTION
**Pull request by numb3r3**

### CODE REVIEW

File: annlite/index.py [hunk 1]
```diff
@@ -747,13 +748,13 @@ def _backup_index_to_remote(self, target_name: str, token: str):
...
             )
 
-        # upload meta table
-        uploader.upload_file(
-            Path(self.data_path) / 'metas.db',
-            target_name=target_name,
-            type='meta_table',
-            cell_id='all',
-        )
+        # # upload meta table
```
Comment (jemmyshin) [inline annlite/index.py:751]: why not upload meta table?

File: annlite/storage/kv.py [hunk 2]
```diff
@@ -111,8 +111,13 @@ def close(self):
                 '`DocStorage` had been closed already, will skip this close operation.'
             )
             return
-        self._db.flush(wait=True)
-        self._db.close()
+        try:
```
Comment (jemmyshin) [inline annlite/storage/kv.py:114]: does this mean that the error is caused by not closing RocksDB correctly?

File: annlite/container.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def __init__(
             CellTable(f'table_{c}', columns=columns) for c in range(n_cells)
         ]
 
-        self._meta_table = MetaTable('metas', data_path=data_path, in_memory=False)
+        self._meta_table = MetaTable('metas', data_path=data_path, in_memory=True)
```
Comment (jemmyshin) [inline annlite/container.py:86]: why put meta table into memory?

Comment (numb3r3) [reply annlite/container.py:86]: This is the source of failure of the delete-then-update issue.

File: annlite/storage/kv.py (hunk 2 above)
Comment (numb3r3) [reply annlite/storage/kv.py:114]: In some cases, the data file is forcefully deleted by another program or `rm -rf`.

File: annlite/index.py (hunk 1 above)
Comment (numb3r3) [reply annlite/index.py:751]: good catch, this is an error I made. I forgot to uncomment them.

File: annlite/storage/kv.py (hunk 2 above)
Comment (numb3r3) [reply annlite/storage/kv.py:114]: oh, i see, the uploading brokes our tests. https://github.com/jina-ai/annlite/actions/runs/4507514031/jobs/7935366275?pr=222 @jemmyshin Do you have any clues about it?

File: annlite/index.py (hunk 1 above)
Comment (numb3r3) [reply annlite/index.py]: oh, i see, the uploading brokes our tests. https://github.com/jina-ai/annlite/actions/runs/4507514031/jobs/7935366275?pr=222 @jemmyshin Do you have any clues about it?

**Pull request merged**